ogre_mesh_viewer.py

```diff
@@ -609,10 +646,18 @@ def shutdown(self):
     import argparse
 
     parser = argparse.ArgumentParser(description="Ogre Mesh Viewer")
-    parser.add_argument("infile", help="path to a ogre .mesh, ogre .scene or any format supported by assimp")
-    parser.add_argument("-c", "--rescfg", help="path to the resources.cfg")
+    parser.add_argument("infile", help="path to a ogre .mesh, ogre .scene or any format supported by assimp", nargs='?')
+    parser.add_argument("-c", "--rescfg", help="path to the resources.cfg", required=False)
+    parser.set_defaults(infile='')
     args = parser.parse_args()
-    app = MeshViewer(args.infile, args.rescfg)
+
+    if args.infile == "":
+        root = tk.Tk()
+        root.withdraw()
+        infile = filedialog.askopenfilename(title="Open", filetypes=[("OGRE Mesh files", "*.mesh"), ("OGRE Scene files", "*.scene")])
```

Comment:
we can also open any of the assimp file types like .obj. You could extend the list for those, but it might change with new assimp releases. Therefore I would just not use any filters here..

Reply:
True, but many times when you export (from Blender for example) to a directory many types of files get dumped there and it can be difficult to discern the types of files that can be opened.

You could have like 10s of png and/or material files, etc. and only one or two meshes and a .scene for example.

Reply:
Maybe just add an "Other (*.*)" filter?